changelog/unreleased/3837

```diff
@@ -0,0 +1,6 @@
+Bugfix: Fix for database migration
+
+For those users who hadn't migrated the camera uploads data yet, the app crashed. This problem
+has been solved and now the app upgrades correctly.
```

Comment:
This is something that users don't know: `For those users who hadn't migrated the camera uploads data yet` It's internal so I would start the summary with something like: `Upgrading from 2.18 or older versions made the app crash due to database migration...`